tljh/configurer.py

```diff
@@ -228,6 +228,8 @@ def update_user_environment(c, config):
     # Set default application users are launched into
     if user_env["default_app"] == "jupyterlab":
         c.Spawner.default_url = "/lab"
+    elif user_env["default_app"] == "classic":
+        c.Spawner.default_url = "/tree"        
```

Comment:
I think this causes this test to fail (https://github.com/jupyterhub/the-littlest-jupyterhub/blob/main/tests/test_configurer.py#L55) by giving it a default _url.